contrib/mesos/pkg/minion/task.go

```diff
@@ -0,0 +1,519 @@
+/*
```

Comment:
To get a proper interface with proper private and public functions, I would move all the task code into a minion/task sub-module.